### DISCUSSION TITLE
Unable to render interface.tags.name in config template

### DISCUSSION THREAD

**Discussion by chuegel** [netbox-community/netbox#13571] [Help Wanted!] [answered]
Hi,

we use a interface tag to render additional interface configurations for our devices but I'm unable to generate the config:

``` 
{%- if interface.tags.name == "no_cdp_lldp" %}
 no cdp enable
 no lldp transmit
 no lldp receive
{%- endif %}
```
Does anybody know how to access the interface tags?

**Comment by candlerb** [comment]
tags is a list.  I would expect something like `{%- if "no_cdp_lldp" in interface.tags %}`, although that depends if 'tags' is a list of strings rather than a list of objects.

Try it in nbshell, and also see https://jinja.palletsprojects.com/en/3.1.x/templates/#other-operators

**Comment by chuegel** [comment]
Thanks for your reply. 
Hmm..this doesn't work

```
<class 'TypeError'>

argument of type '_TaggableManager' is not iterable

Python version: 3.10.12
NetBox version: 3.5.8
Plugins: None installed
```

**Comment by candlerb** [comment, marked as answer]
Try `interface.tags.names()`

```
/opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py nbshell
### NetBox interactive shell (netbox3)
### Python 3.8.10 | Django 4.1.10 | NetBox 3.5.8
### lsmodels() will show available models. Use help(<model>) for more info.
>>> d = Device.objects.get(name="nuc1")
>>> d.tags.names()
<RestrictedQuerySet ['prom_node']>
>>> "prom_node" in d.tags.names()
True
```

**Comment by chuegel** [comment]
yes, that worked. Thank  you very much.
